Clarify path matching behavior in Express.js documentation (#2036)
expressjs/expressjs.com: _includes/api/en/4x/app-use.md

<tr>
<td>Path</td>
<td markdown="1">

> This will match the exact path `/abcd` and any paths beginning with `/abcd/` (including subdirectories), but will not match paths like `/abcde` :

Reviewer: By saying that it’s exact, it already indicates that no other routes different from /abcd will be added to the middleware.

```suggestion
Matches the exact path `/abcd` and any sub-paths starting with `/abcd/` (for example, `/abcd/foo`):
```
